src/protocol.h

```diff
@@ -32,91 +32,106 @@
-#define PROTMESSID_REQ_NETW_TRANSPORT_PROPS   21 // request properties for network transport
-#define PROTMESSID_DISCONNECTION              22 // OLD (not used anymore)
-#define PROTMESSID_REQ_CHANNEL_INFOS          23 // request channel infos for fader tag
-#define PROTMESSID_CONN_CLIENTS_LIST          24 // channel infos for connected clients
-#define PROTMESSID_CHANNEL_INFOS              25 // set channel infos
-#define PROTMESSID_OPUS_SUPPORTED             26 // tells that OPUS codec is supported
-#define PROTMESSID_LICENCE_REQUIRED           27 // licence required
-#define PROTMESSID_REQ_CHANNEL_LEVEL_LIST     28 // OLD (not used anymore) // TODO needed for compatibility to old servers >= 3.4.6 and <= 3.5.12
-#define PROTMESSID_VERSION_AND_OS             29 // version number and operating system
-#define PROTMESSID_CHANNEL_PAN                30 // set channel pan for mix
-#define PROTMESSID_MUTE_STATE_CHANGED         31 // mute state of your signal at another client has changed
-#define PROTMESSID_CLIENT_ID                  32 // current user ID and server status
-#define PROTMESSID_RECORDER_STATE             33 // contains the state of the jam recorder (ERecorderState)
-#define PROTMESSID_REQ_SPLIT_MESS_SUPPORT     34 // request support for split messages
-#define PROTMESSID_SPLIT_MESS_SUPPORTED       35 // split messages are supported
+#define PROTMESSID_ILLEGAL 0                 // illegal ID
+#define PROTMESSID_ACKN 1                    // acknowledge
+#define PROTMESSID_JITT_BUF_SIZE 10          // jitter buffer size
+#define PROTMESSID_REQ_JITT_BUF_SIZE 11      // request jitter buffer size
+#define PROTMESSID_NET_BLSI_FACTOR 12        // OLD (not used anymore)
+#define PROTMESSID_CHANNEL_GAIN 13           // set channel gain for mix
+#define PROTMESSID_CONN_CLIENTS_LIST_NAME 14 // OLD (not used anymore)
+#define PROTMESSID_SERVER_FULL 15            // OLD (not used anymore)
+#define PROTMESSID_REQ_CONN_CLIENTS_LIST 16  // request connected client list
+#define PROTMESSID_CHANNEL_NAME 17           // OLD (not used anymore)
+#define PROTMESSID_CHAT_TEXT 18              // contains a chat text
+#define PROTMESSID_PING_MS 19                // OLD (not used anymore)
+#define PROTMESSID_NETW_TRANSPORT_PROPS 20   // properties for network transport
+#define PROTMESSID_REQ_NETW_TRANSPORT_PROPS                                    \
+    21                              // request properties for network transport
+#define PROTMESSID_DISCONNECTION 22 // OLD (not used anymore)
+#define PROTMESSID_REQ_CHANNEL_INFOS 23 // request channel infos for fader tag
+#define PROTMESSID_CONN_CLIENTS_LIST 24 // channel infos for connected clients
+#define PROTMESSID_CHANNEL_INFOS 25     // set channel infos
+#define PROTMESSID_OPUS_SUPPORTED 26    // tells that OPUS codec is supported
+#define PROTMESSID_LICENCE_REQUIRED 27  // licence required
+#define PROTMESSID_REQ_CHANNEL_LEVEL_LIST                                      \
```

Comment:
I particularly don't like the spacing here.  If it's going to split, keep the `\` tight to the end of the useful content, not off the side where it could be missed when reading.

Reply:
fixed by adding
```
AlignEscapedNewlines: DontAlign
```